src/duplicate-search/duplicate-search-controls.module.css

```diff
@@ -0,0 +1,53 @@
+@import '../variables.css';
+
+.statusFilterWrapper {
+	width: 16rem;
+}
+
+.statusFilterControl {
+	border: 1px solid var( --color-border-dark );
+	border-radius: 2px;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	overflow: hidden; /* Hide the border radius on the buttons */
+}
+
+.statusFilterControl:focus-within {
+	border-color: var( --color-primary );
```

Comment:
👋 Hey @dpasque, sorry for chiming in late. 

**nitpick (if-minor):** Super nitpick that might have been a conscious decision, but the border for the focused state actually increases to `1.5px` in the designs. I don't know if `outline` is less desirable, but it seems to add thickness without increasing the size of the component:
```
    border-color: var( --color-primary );
    outline: 0.5px solid var( --color-primary );
```

What do you think?

Reply:
Ack, sorry I merged too early! I had been mostly focused on the color change, so went ahead after Karens review, my bad!

Oh sure, that's a great idea! 😄  I can spin up a new PR with the fixes!

My main question on this bit: are you okay if we just make it 1px? I'm not a huge fan of half pixels in CSS... they're rendered differently by different browsers and screen densities, and I think it's often simpler to avoid them if possible! Does that throw off the design mojo too much?

Reply:
> Ack, sorry I merged too early! I had been mostly focused on the color change, so went ahead after Karens review, my bad!

No worries!

> My main question on this bit: are you okay if we just make it 1px?

I have no problem with that! I didn't know that about half pixels. I'll make the change in the Figma file as well. Thanks for all of your expertise on this 🙇

Reply:
Got a fix PR started here: https://github.com/Automattic/bugomattic/pull/72